app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -71,6 +74,7 @@ sealed class NavigationItem(
         const val HOME_START_TAB_ARGUMENT: String = "start_tab_index"
         const val CONVERSATION_ID_ARGUMENT: String = "conversation_id"
 
+        @ExperimentalMaterialApi
```

Comment:
Use `@OptIn(ExperimentalMaterialApi::class)`, this way you don't have to duplicate it for every parent